src/main/scala/se/nimsa/sbx/seriestype/SeriesTypeDAO.scala

```diff
@@ -165,8 +167,11 @@ class SeriesTypeDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: Execut
   }.map(_ => Unit)
 
   def upsertSeriesSeriesType(seriesSeriesType: SeriesSeriesType): Future[SeriesSeriesType] = db.run {
-    seriesSeriesTypes.insertOrUpdate(seriesSeriesType)
-  }.map(_ => seriesSeriesType)
+    seriesSeriesTypes += seriesSeriesType
+  }.map(_ => seriesSeriesType).recover {
+    case _: SQLIntegrityConstraintViolationException => seriesSeriesType //Duplicate key -> series already tagged with series type
```

Comment:
If I understand this correctly, this exception is thrown when using MySql but not when using H2?

Reply:
Test fails in MetaDataRoutesTest when adding same series type twice.

Reply:
I suppose this is thrown by the connector so using the broader SQLException might be necessary to cover differences among connectors.